lib/tools/system-calls.js

```diff
@@ -438,7 +438,9 @@ systemCallMethods.launchAVD = async function (avdName, avdArgs, language, countr
   let proc = new SubProcess(emulatorBinaryPath, launchArgs);
   await proc.start(0);
   proc.on('output', (stdout, stderr) => {
-    log.info(`[AVD OUTPUT] ${stdout || stderr}`);
+    for (let line of (stdout || stderr || '').split('\n').filter(Boolean)) {
```

Comment:
I don't know if there would ever be a case when `stdout` and `stderr` are both empty. But this adds no complexity, and no decrease in performance (since `||` short-circuits), while making sure that the call to `split` will not fail.